### PR TITLE
Adds toSeverity humanizer function

### PR DESCRIPTION
**Pull request by Lutie**
Adds a function which returns a term according to a given severity score.
The term in question is the same as those given here for example: https://www.first.org/cvss/calculator/3.1

### CODE REVIEW

File: src/humanizer.ts [hunk 1]
```diff
@@ -49,3 +49,18 @@ export const humanizeBaseMetricValue = (
       return 'Unknown';
   }
 };
+
+/**
+ * Stringify an score into a severity string ('None' | 'Low' | 'Medium' | 'High' | 'Critical')
+ * @param score
+ */
+export const toSeverity = (score: number): Severity =>
```
Comment (pmstss) [inline src/humanizer.ts:57]: i would suggest name change to be aligned with other humanizer functions

```suggestion
export const humanizeScore = (score: number): string=>
```

File: src/humanizer.ts [hunk 2]
```diff
@@ -49,3 +49,18 @@ export const humanizeBaseMetricValue = (
       return 'Unknown';
   }
 };
+
+/**
+ * Stringify an score into a severity string ('None' | 'Low' | 'Medium' | 'High' | 'Critical')
```
Comment (pmstss) [inline src/humanizer.ts:54]: specification calls it as "qualitative severity rating"

```suggestion
 * Stringify a score into a qualitative severity rating string
```

File: src/models.ts [hunk 3]
```diff
@@ -9,6 +9,8 @@ export enum BaseMetric {
   AVAILABILITY = 'A'
 }
 
+export type Severity = 'None' | 'Low' | 'Medium' | 'High' | 'Critical';
```
Comment (pmstss) [inline src/models.ts:12]: it's not required in the models, as used only in humanizer

```suggestion
```

File: tests/humanizer.spec.ts [hunk 4]
```diff
@@ -2,7 +2,8 @@ import {
   BaseMetric,
   BaseMetricValue,
   humanizeBaseMetric,
-  humanizeBaseMetricValue
+  humanizeBaseMetricValue,
+  toSeverity
```
Comment (pmstss) [inline tests/humanizer.spec.ts:6]:
```suggestion
  humanizeScore
```

File: tests/humanizer.spec.ts [hunk 5]
```diff
@@ -53,3 +54,22 @@ describe('humanizer', () => {
...
+  });
+  it('Should give Medium risk level when score is below 6', () => {
+    expect(toSeverity(6)).to.equal('Medium');
+    expect(toSeverity(4.2654)).to.equal('Medium');
+  });
+  it('Should give High risk level when score is below 8.5', () => {
+    expect(toSeverity(8.5)).to.equal('High');
+    expect(toSeverity(7.2654)).to.equal('High');
+  });
+});
```
Comment (pmstss) [inline tests/humanizer.spec.ts:75]: Suggested tests changes:
- move them to existing humanizer test suite
- wording changes
- tests for boundary values
- tests for "Critical" qualitative severity rating
- test only valid score values

```suggestion

  it('should humanize score as "None" for zero value', () => {
    expect(humanizeScore(0)).to.equal('None');
  });

  it('should humanize score as "Low" for values from interval [0.1, 3.9]', () => {
    expect(humanizeScore(0.1)).to.equal('Low');
    expect(humanizeScore(1.2)).to.equal('Low');
    expect(humanizeScore(3.9)).to.equal('Low');
  });

  it('should humanize score as "Medium" for values from interval [4.0, 6.9]', () => {
    expect(humanizeScore(4.0)).to.equal('Medium');
    expect(humanizeScore(4.2)).to.equal('Medium');
    expect(humanizeScore(6.9)).to.equal('Medium');
  });

  it('should humanize score as "High" for values from interval [7.0, 8.9]', () => {
    expect(humanizeScore(7.0)).to.equal('High');
    expect(humanizeScore(7.5)).to.equal('High');
    expect(humanizeScore(8.9)).to.equal('High');
  });

  it('should humanize score as "Critical" for values from interval [9.0, 10.0]', () => {
    expect(humanizeScore(9.0)).to.equal('Critical');
    expect(humanizeScore(9.5)).to.equal('Critical');
    expect(humanizeScore(10.0)).to.equal('Critical');
  });
});
```

**Review by pmstss — CHANGES_REQUESTED** [review]
thanks @Lutie for good proposal, please have a look at suggested improvements

**Comment by Lutie** [comment]
> thanks @Lutie for good proposal, please have a look at suggested improvements

These are excellent suggestions for improvements, I took them into consideration, thank you!

**Comment by pmstss** [comment]
@Lutie please cleanup removed `Severity` from `import { BaseMetric, BaseMetricValue, Severity } from './models';`

**Review by pmstss — APPROVED** [review]

**Review by derevnjuk — APPROVED** [review]
